README.md

```diff
@@ -1,75 +1,68 @@
-# KintoHub {name}-example
+# KintoHub Spring Boot Example
 
 ## Overview
-TODO: Cool description / goal of what technology you are bringing onto the plate with what language. This should be sweet and concise.
 
-[Live Example](live/example/on/kintohub)
+Spring Boot makes it easy to create stand-alone, production-grade Spring based Applications that you can "just run". It can be built easily for microservice on KintoHub!
+
+[Live Example](https://8caf94bb-c2e5-45cf-898d-ca5b19954cda.api.master.kintohub.com/springboot-example/hello/world)
 
 __About KintoHub:__
 
 KintoHub aligns teams to ship & operate cloud native apps with ease. [Learn More](https://www.kintohub.com)
-## [Optional] Requirements
-
-- Ensure you have an account on [KintoHub](www.kintohub.com/sign-up)
-- [Optional] Required Kintohub features: [list features] (like pro sub features)
-- [Optional] specific packages needed for project to run
 
 ## Deployment
-1. Apply this template to your [Github](https://github.com/kintohub/example-template/generate)
-2. Create a [KintoBlockType](www.link.to.create.kintoblock.type.here) and include:
-
-You're now good to build! Click `Build Latest Commit`. Once complete, Click  Now click `Add To Project`.
-
-3. Create Project and include:
-
+1. Apply this template to your [Github](https://github.com/kintohub/springboot-example/generate)
+2. Create a [Microservice Block](https://docs.kintohub.com/docs/kintoblocks/microservices) on KintoHub
+3. Connect your GithubApp to KintoHub and select the Repository you just generated on your account.
+4. Set the **name** of your kinto block
+5. Choose `Java` as the **language** and `openjdk-8-alpine` as the **version**
+6. Set the **Build Command** as `./mvnw clean package`
+7. Set the **Start Command** as `java -jar ./target/springboot-example-1.0.0.jar`
+8. Set the **Port** as `80`
 
-You're ready to go live Click `Deploy`
+You're now good to go! Click **Create Microservice**. Now click **Build Latest Commit**
 
-## Build & Deployment Configurations
-Use the following examples for build or deployment environment variables:
+Once complete, Click **Add to Project** 
+NOTE: Please assign at least **256MB** memory for your Java microservice block during the deployment.
```

Comment:
I would mention clear steps for this actually by clicking the settings icon (maybe take a screenshot and include it in the markdown?)

Reply:
I would prefer an external link to our doc.